### ISSUE TITLE
Feature request: Inject plugin version for lifecycle goal

### ISSUE THREAD

**Issue by niriak** [release-engineering/pom-manipulation-ext#999]
According to the documentation, plugin versions can only be overridden but not created.
It would be nice if the following would also work, even if there is no entry in the _pom.xml_:

`mvn deploy -DpluginOverride.org.apache.maven.plugins:maven-deploy-plugin@*=3.1.1`

This is to force the use of the _maven-deploy-plugin_ version 3.1.1, regardless of whether there is a definition within the pom, parent pom etc or not.

**Comment by rnc** [comment]
I don't think this is the typical use-case for PME. You could inject a definition of the deploy plugin? Is there a particular reason you wanted to do this (or even to use the deploy plugin 3.1.1 - is it due to conflict with the different formats for `altDeploymentRepository` with older versions? )

**Comment by rnc** [comment]
Slightly off topic, but if the problem is with the altDeploymentRepository and the change of format you may be interested in https://github.com/rnc/alt-deploy-maven-extension
